src/styles/tokens.css

```
@@ -0,0 +1,62 @@
+  --color-state-overlay: oklch(0% 0 0 / 0.06);
+  --color-state-disabled: oklch(0% 0 0 / 0.12);
+
+  /* system */
+  --color-success: oklch(70% 0.149 163);
+  --color-warning: oklch(84% 0.164 84);
+  --color-error: oklch(64% 0.208 25);
+  --color-info: oklch(62% 0.188 260);
+
+  /* typography */
+  --font-family-sans: vw_textregular, system-ui, ui-sans-serif;
+  --font-family-novamono: Nova Mono;
```

Stylelint (17.12.0)

[error] Expected "Nova" to be "nova" (value-keyword-case)

(value-keyword-case)